js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/AutomaterializationRoot.tsx

```diff
@@ -203,8 +204,12 @@ const GlobalAutomaterializationRoot = () => {
         <Subtitle2>Evaluation timeline</Subtitle2>
       </Box>
       {!data ? (
-        <Box padding={{vertical: 48}}>
-          <Spinner purpose="page" />
```

Comment:
Unrelated, but I don't like these "page" level spinners and I'm removing them whenever I find them.

Reply:
We need a loading state though?